CONTRIBUTING.md

```diff
@@ -145,11 +145,9 @@ Also do your best to factor commits appropriately, not too large with unrelated
 
 ### Contributor License Agreement
 
-You must sign a [.NET Foundation Contribution License Agreement (CLA)](https://cla.dotnetfoundation.org) before your PR will be merged. This is a one-time requirement for projects in the .NET Foundation. You can read more about [Contribution License Agreements (CLA)](http://en.wikipedia.org/wiki/Contributor_License_Agreement) on Wikipedia.
+You must sign a .NET Foundation Contribution License Agreement (CLA) before your PR will be merged. This is a one-time requirement for projects in the .NET Foundation. You can read more about [Contribution License Agreements (CLA)](http://en.wikipedia.org/wiki/Contributor_License_Agreement) on Wikipedia and the [Microsoft Open Source](https://opensource.microsoft.com/cla/) program.
```

Comment:
https://cla.dotnetfoundation.org opens a nice .pdf rendering of the .NET Foundation CLA now. Can we add the link back?

Reply:
😆 it didn't when I checked a couple days ago, and the issued I filed at https://github.com/dotnet-foundation/wg-maintainers/issues/91 still hadn't been updated, so I gave up. (sigh / facepalm).